packages/@angular/cli/blueprints/ng2/files/__path__/tsconfig.json

```diff
@@ -4,9 +4,12 @@
     "declaration": false,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "lib": ["es6", "dom"],
+    "lib": [
+      "es2015",
```

Comment:
@delasteve should it be `"es2015"` here (lib and module properties) when all other files are `"es2016"`?